drizzlepac/outputimage.py

```diff
@@ -530,32 +545,40 @@ def writeFITS(self, template, sciarr, whtarr, ctxarr=None,
                 # header to Primary header...
                 if dqhdr:
                     for _card in dqhdr.cards:
-                        if ( (_card.keyword not in RESERVED_KEYS) and
+                        if ((_card.keyword not in RESERVED_KEYS) and
```

Comment:
This adds to the argument that `hdu` cannot contain `'EXTNAME'` and `'EXTVER'` because they are not copied over.

Reply:
Leaving this in only insures against any unexpected problems with the inputs, and does not hurt the final code or take any measurable time to process.  So, this line was not modified in commit 4b565a7.